torchmetrics/detection/map.py

```diff
@@ -190,6 +190,42 @@ class MAP(Metric):
             Callback that performs the allgather operation on the metric state. When ``None``, DDP
             will be used to perform the allgather
 
+    Example:
+        >>> from torchmetrics.detection.map import MAP
```

Comment:
@Borda can you help here?
I added the example to showcase the new import path but this makes some test fail due to not having the correct version of torchvision installed that is nessesary for this metric

Reply:
seems I found some workaround if the output is different, but it does not work in the method raises an exception...
https://stackoverflow.com/questions/42708809/python-doctest-skip-a-test-conditionally